src/fetcher.js

```diff
@@ -37,40 +36,36 @@ export class Fetcher {
   }
 
   /**
-   * Removes the last element of the url templates filters array.
-   * @return {void}
+   * Removes the latest URL template filter added.
    */
-  removeURLTemplateFilter() {
+  removeLastURLTemplateFilter() {
     this.URLTemplateFilters.pop();
   }
 
   /**
-   * Returns the number of filters of the url templates filters array.
+   * Returns the number of URL template filters added.
    * @return {Number}
    */
   countURLTemplateFilters() {
     return this.URLTemplateFilters.length;
   }
 
   /**
-   * Removes all the filter functions from the url templates filters array.
-   * @return {void}
+   * Removes all the URL template filters added.
    */
   clearURLTemplateFilters() {
     this.URLTemplateFilters = [];
   }
 
   /**
    * Fetches a VAST document for the given url.
-   * Returns a Promise which resolves,rejects according to the result of the request.
-   * @param {String} url - The url to request the VAST document.
-   * @param {Number} wrapperDepth - How many times the current url has been wrapped.
-   * @param {String} previousUrl - Url of the previous VAST.
-   * @param {Object} wrapperAd - Previously parsed ad node (Wrapper) related to this fetching.
-   * @param {Number} maxWrapperDepth - The maximum number of Wrapper that can be fetch
-   * @param {Function} emitter - The function used to Emit event
-   * @emits  VASTParser#VAST-resolving
-   * @emits  VASTParser#VAST-resolved
```

Comment:
We should keep thoses informations, aswell as `Returns a Promise which resolves,rejects according to the result of the request.`